src/components/ImgEditor/useImgEditor.ts

```diff
@@ -74,14 +74,7 @@ export default function useImgEditor<T extends FieldValues, K extends Path<T>>({
   };
 
   function handleCropResult(blob: Blob | null, originalFile: File) {
-    if (!blob) {
-      return showModal(Prompt, {
-        type: "error",
-        headline: "Edit image",
-        title: "Failed to crop image",
-        children: GENERIC_ERROR_MESSAGE,
-      });
-    }
+    if (!blob) return handleError({ context: "cropping image" });
```

Comment:
```suggestion
    if (!blob) return handleError("Blob can't be `null`", { context: "cropping image" });
```
`{ context }` is part of the 2nd argument.